sdks/python/apache_beam/examples/complete/game/user_score.py

```diff
@@ -159,7 +158,7 @@ def run(argv=None, save_main_session=True):
   parser.add_argument(
       '--input',
       type=str,
-      default='gs://apache-beam-samples/game/gaming_data*.csv',
```

Comment:
This prevented this single-file example from running. Thanks, @sirenbyte for pointing out!

Reply:
Looks like this is failing in CI - https://github.com/apache/beam/actions/runs/3760568592/jobs/6391627592

Reply:
Found the proper bucket for this example: `gs://apache-beam-samples/game/small/gaming_data.csv`